pyvista/core/filters.py

```diff
@@ -1156,7 +1156,7 @@ def warp_by_vector(dataset, vectors=None, factor=1.0, inplace=False):
             field, vectors = dataset.active_vectors_info
         arr, field = get_array(dataset, vectors, preference='point', info=True)
         # check that this is indeed a vector field
-        if field != pyvista.POINT_DATA_FIELD or arr.ndim != 2 or arr.shape[1] != 3 :
```

Comment:
This is weird: I've copied that part from warp_by_scalar but it failed the tests (maybe this is not tested in warp_by_scalar?). I'm therefore removing it.